query/filter/comparison.go

```diff
@@ -22,8 +22,11 @@ import (
 )
 
 const (
-	EQ = "$eq"
-	GT = "$gt"
+	EQ  = "$eq"
+	GT  = "$gt"
+	LT  = "$lt"
+	GTE = "$gte"
+	LTE = "$lte"
```

Comment:
need a `$ne` not equals matcher as well.

Reply:
will do it iteratively, need to test on TS as well.